src/api/component.js

```diff
@@ -2,12 +2,15 @@ import { patchInner } from 'incremental-dom';
 import {
   connected as $connected,
   created as $created,
+  ctor_props as $ctor_props,
```

Comment:
nit: it's preferable to not have underscores in `NON_UPPERCASE_CONSTANT` names. If you can `camelCase` it, that'd be awesome.